Documentation/ColumnsConfig/CommonProperties/Mm.rst

```diff
@@ -186,16 +190,14 @@ Analyzer:
 Columns of the intermediate MM table
 ------------------------------------
 
-The intermediate table has three columns as a minimum:
-
-uid\_local, uid\_foreign (Required)
-   Storing uids of both sides. If done right, this is reflected in the table name - :code:`tx_foo_local_foreign_mm`
+The intermediate table may have the following fields:
 
-sorting (Required)
-   Is a required field used for ordering the items.
+uid\_local, uid\_foreign
+   Storing uids of both sides. If done right, this is reflected in the table
+   name - :code:`tx_foo_local_foreign_mm`
 
-sorting\_foreign
-   Is required if the relation is bidirectional (see description and example below table)
+sorting, sorting\_foreign
+   Are required fields used for ordering the items.
```

Comment:
This is changing the semantics: are the fields now always required?

Reply:
yes